pkg/controller/policyrecommendation/policyrecommendation_controller.go

```diff
@@ -471,13 +471,14 @@ func (r *ReconcilePolicyRecommendation) Reconcile(ctx context.Context, request r
 		return reconcile.Result{}, err
 	}
 
+	targetNamespace := helper.InstallNamespace()
+
 	setUp := render.NewSetup(&render.SetUpConfiguration{
```

Comment:
@gantony do we need to call `NewSetup` at all here? 

`NewSetup` seems to do this:

- Create a namespace (if configured)
- Copy pull secrets
- Create a secrets binding for RBAC

When running in calico-system, these should all already be done for us, right?

Same would go when deploying into an existing tenant namespace?